templates/CRM/MembershipExtras/Page/NextPeriodTab.tpl

```diff
@@ -1,5 +1,4 @@
 <script>
-var recurringContributionID = {$recurringContributionID};
 var financialTypes = JSON.parse('{$financialTypes|@json_encode}');
 var membershipTypes = JSON.parse('{$membershipTypes|@json_encode}');
```

Comment:
Same thing applies here as well.
`var membershipTypes = {$membershipTypes|@json_encode};` should do